packages/fetchai/connections/p2p_libp2p_client/connection.py

```diff
@@ -229,7 +229,7 @@ async def receive(self, *args, **kwargs) -> Optional["Envelope"]:
                     self._connection_status.is_connected
                     or self._connection_status.is_connecting
                 ):
-                    await self.disconnect()
+                    await self.disconnect()  # pragma: no cover
```

Comment:
on my local machine, with file (missing from develop) `test_errors.py`, this is already covered by  `tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py::TestLibp2pClientConnectionNodeDisconnected::test_node_disconnected`
(https://github.com/fetchai/agents-aea/blob/feature/libp2p-improve-coverage/tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py#L128)